packages/opentelemetry-plugin-http/src/utils.ts

```diff
@@ -242,4 +242,15 @@ export class Utils {
     const type = typeof options;
     return type === 'string' || (type === 'object' && !Array.isArray(options));
   }
+
+  /**
+   * Check whether the given request should be ignored
+   * Use case: Typically, exporter `SpanExporter` can use http module to send spans.
+   * This will also generate spans (from the http-plugin) that will be sended through the exporter
+   * and here we have loop.
+   * @param {RequestOptions} options
+   */
+  static isOpenTelemetryRequest(options: RequestOptions) {
```

Comment:
More of a style thing and not for this PR, but my recommendation is generally to just export utility functions as `export function(...` rather than putting them in a container class, since the module itself already acts as a container.

Reply:
if you open an issue, I will change this style thing.

Reply:
Sure, I just opened https://github.com/open-telemetry/opentelemetry-js/issues/345 to track, thanks!